src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(),new Random().nextInt(100)).toString();
```

Comment:
let's create constant for 100 number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names